contracts/_interfaces/ILayer2BridgeGateway.sol

```diff
@@ -0,0 +1,56 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
How is this different from IBridgeGateway?

Reply:
Open to suggestions on better names. 

However, the distinction I'm trying to draw is between homogeneous layer 2 bridges (like the maitc bridge) and heterogenous bridges (like meter passport). In the latter's case the contracts on ethereum don't have visibility into the recipient's address as the address format could be different.  

The only difference is `ILayer2BridgeGateway` additionally logs the `recipientInTargetChain` ..

Reply:
Ok that makes more sense. This would be useful in the file documentation here.